src/services/PollingBackend.js

```diff
@@ -124,8 +124,10 @@ class PollingBackend {
 
 		this.#pollActive = true
 
-		const autosaveContent = shouldSave ? this.#syncService._getContent() : null
-		const documentState = shouldSave ? this.#syncService.getDocumentState() : null
+		const shouldAutosave = true // FIXME: Figure out when we should autosave
```

Comment:
- [ ] For this PR: introduce a time check here to only send this every 30 seconds
- Follow up: 
  - Move save request out of the sync request to a dedicated endpoint
- Maybe later: 
  - Maybe not directly write the content to the file but put it in cache/db for persisting later if all connections are closed #1042 
  - See if we can notify other clients about autosave via awareness messages/simple document state